decidim-accountability/spec/services/decidim/accountability/searchable_result_resource_spec.rb

```diff
@@ -121,7 +121,7 @@ def expect_searchable_resource_to_correspond_to_result(searchable, result, local
       attrs.delete("id")
       attrs.delete("created_at")
       attrs.delete("updated_at")
-      expect(attrs.delete("datetime").to_s(:short)).to eq(result.start_date.in_time_zone.to_s(:short))
+      expect(attrs.delete("datetime").to_fs(:short)).to eq(result.start_date.in_time_zone.to_fs(:short))
```

Comment:
There are two new related rubocop offenses that introduced these changes: `Rails/ToSWithArgument` and `Rails/ToFormattedS`. These are automated corrections.